keps/sig-storage/20191125-bucket-provisioning.md

```diff
@@ -0,0 +1,420 @@
+---
+mmrrtitle: Object Bucket Provisioning
+authors:
+  - "@jeffvance"
+  - "@copejon"
+owning-sig: "sig-storage"
+reviewers:
+  - "@saad-ali"
+  - "@alarge"
+  - "@erinboyd"
+  - "@guymguym"
+  - "@travisn"
+approvers:
+  - TBD
+editor: TBD
+creation-date: 2019-11-25
+last-updated: 2020-04-28
+status: provisional
+---
+
+# Object Bucket Provisioning
```

Comment:
There's an error that says "error validating KEP metadata: missing key title".
This should be:
```
# Title

Object Bucket Provisioning
```

Reply:
@xing-yang I made the changes to title but still get validation err. I think I misunderstand your comment.